content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Troubleshooting CNI plugin-related errors
+content_type: task
+reviewers:
+- mikebrow
+weight: 10
+---
+
+<!-- overview -->
+
+To avoid CNI plugin-related errors, verify that you are using or upgrading to a
+container runtime that has been tested to work correctly with your version of
+Kubernetes.
+
+For example, the following container runtimes have been tested with Kubernetes
+v1.24:
+
+* containerd v1.6.4 and later, v1.5.11 and later
+* CRI-O v1.24.0 and later
```

Comment:
Similar case with this version, as well. We can mention that there are plans around eliminating this. in future versions, but we cannot outright claim to have tested them against v1.24.

Reply:
fyi.. not sure how you want to word this.. cri-o's versioning is aligned with the version of kubernetes 

cri-o v1.23.x is for kubernetes v1.23.x
cri-o v1.24.x is for kubernetes v1.24.x  

https://github.com/cri-o/cri-o#compatibility-matrix-cri-o--kubernetes